src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+/* #region card */
+.card {
+
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  --main-color: #060b35;
+  --second-color: #616070;
+  --blue-color: #00acdc;
+}
+
+.card__product-image {
+  width: 160px;
+  height: 134px;
+  object-fit: cover;
+  margin: 0 auto 40px;
+}
+
+.card__product-tile {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--main-color);
+  margin-bottom: 4px;
+}
+
+h1 {
+  font-style: normal;
+}
```

Comment:
Use class to style elements, or if u make reset styles, move styles to the start of document or to separate file